### DISCUSSION TITLE
⛔ 🔗 Video Element Improvements (Duplicate)

### DISCUSSION THREAD

**Discussion by heroicP** [elementor/elementor#18937] [Feature Request]
Hi,

I get frustrated when trying to use the default Elementor Video Element. Usually it's so bad I just end up using a section with an icon element to launch a lightbox video. Here are just a couple things it seems like the video element should do so it can actually be used out of the box.

Custom play button: Can the ability to add your own custom SVG play button be added? The default one just doesn't match the branding of every single site I build.

Hover state: It would be great if we could set some sort of animation when a user hovers a video (that's set to open in a lightbox)

**Comment by nicholaszein** [comment]
Hello there!

thanks for submitting your feature request! 🙏 

### 🗃️ Please understand that while we do welcome new feature requests, we can't track multiple requests per thread. 

Please follow our guidelines for submitting a new [🚀 Feature Request](https://github.com/elementor/elementor/discussions/new?category=feature-request), and submit only ☝️ **_<ins>one request per thread</ins>_** so we can track them better and allow users to vote only for the features they want to see implemented.

Kind regards
